meta-mbl-distro/scripts/create-update-payload

```diff
@@ -0,0 +1,191 @@
+#!/usr/bin/env python3
+# Copyright (c) 2019 Arm Limited and Contributors. All rights reserved.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
+
+"""
+This script will create a bootloader update payload tar archive.
+
+You can specify a compression type for the tar archive. The default is gz and
+the available options are gz, bz2 and xz.
+
+To use the script you can either pass in a space separated list of file names
+or a list of bootloader slot numbers. We then map the file names of the update
+components to the corresponding bitbake variables, which are named according
```

Comment:
I'm not sure the stuff about BitBake variables should be in this comment - it's an implementation detail. Perhaps explaining what the "bootloader slot numbers" are would be more useful.